src/wallet/db.h

```diff
@@ -220,6 +220,7 @@ enum class DatabaseStatus {
     FAILED_LOAD,
     FAILED_VERIFY,
     FAILED_ENCRYPT,
+    FAILED_INVALID_BACKCUP_FILE,
```

Comment:
In 4053a210a6126f85779e5ccab248d13a74d27897 "refactor: Move restorewallet() RPC logic to the wallet section"

nit: typo

```suggestion
   FAILED_INVALID_BACKUP_FILE,
```

Reply:
Fixed in abbb7ec